Templates/Per Tenant Extension/.github/workflows/AddNewTestApp.yaml

```diff
@@ -3,11 +3,11 @@ name: Add new test app
 on:
```

Comment:
To stick with the terminology, I guess this action should be "Create a new test app"